modules/virt-about-vms-and-boot-sources.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-and-using-boot-sources.adoc
+// * virt/vm_templates/virt-creating-vm-template.adoc
+
+[id="virt-about-vms-and-boot-sources_{context}"]
+
+= About virtual machines and boot sources
+
+Virtual machines consist of a virtual machine definition and one or more disks, backed by data volumes. Virtual machine templates enable you to create virtual machines using predefined virtual machine specifications.
+
+To use the boot sources feature, install the latest release of {VirtProductName}. The namespace `openshift-virtualization-os-images` enables the feature and is installed with the {VirtProductName} Operator.
+Once the boot source feature is installed, you can create boot sources, attach them to templates, and create virtual machines from the templates. 
+
+Every virtual machine template requires a boot source, which is a fully configured virtual machine disk image including configured drivers. Each virtual machine template contains a virtual machine definition with a pointer to the boot source. Each boot source has a predefined name and namespace. For some operating systems, a boot source is automatically provided. If it is not provided, then a custom boot source must be prepared by the administrator.
+
+After preparing and uploading boot source image files, you create configured virtual machines by attaching the boot source to a virtual machine template.
```

Comment:
This sentence seems redundant since you mention the same thing in the next paragraph.

Reply:
yes, done.